frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```diff
@@ -382,7 +382,7 @@ const allHostTableHeaders: IDataColumn[] = [
       return <HeaderCell value={titleWithToolTip} disableSortBy />;
     },
     disableSortBy: true,
-    accessor: "mdm_enrollment_status",
+    accessor: "mdm_enrollment_status", // TODO: should that be "mdm.enrollment_status"? I don't see other nested accessor as examples...
```

Comment:
This should be confirmed by the frontend, but I believe the react-table way to do this would be to use an optional id  along with the accessor:
```
accessor: "mdm.enrollment_status",
id: "mdm_enrollment_status",
```

Reply:
I think we'd need to test to confirm, but I believe just: 

`accessor: "mdm.enrollment_status",`

would work, not sure it needs the `id`. If that doesn't, an alternative would be: 

```javascript
    accessor: (originalRow) => {
        return originalRow.mdm.enrollment_status;
    },
```

I found a discussion about this here: https://github.com/TanStack/table/issues/579#issuecomment-841844029

Reply:
Yeah, I also tested the dot notation for the accessor and it works. I'm suggesting the `id` too because the the CSS class is generated from the column [id](https://github.com/fleetdm/fleet/blob/a5a6e3ca23869b1af91b161c0b2937b0112ab3b5/frontend/components/TableContainer/DataTable/DataTable.tsx#L515) (which defaults to the accessor if id is not specified). I didn't want the extra dot interpolated in the CSS selector (see comment [below](https://github.com/fleetdm/fleet/pull/9505#discussion_r1087285600)).

Reply:
> I didn't want the extra dot interpolated in the CSS selector 

Smart thinking! Thanks.

Reply:
Updated as suggested with `id`.